client/config_parser.py

```diff
@@ -103,6 +109,10 @@ def parse_config(self, config_file: str = 'config.yaml') -> TranslationConfig:
             else translation_config_input[self.__INPUT_DIR]
         config.output_directory = self.__DEFAULT_OUTPUT_DIR if self.__OUTPUT_DIR not in translation_config_input \
             else translation_config_input[self.__OUTPUT_DIR]
+        config.default_database = self.__DEFAULT_DEFAULT_DATABASE if self.__DEFAULT_DATABASE not in translation_config_input \
```

Comment:
I think the default_database and schema_search_path are optional fields in the API.
Maybe only assign them if these fields showed up in the config.yaml file.